apps/web-giddh/src/app/invoice/invoice.component.html

```diff
@@ -55,7 +55,7 @@
 
                     <!--<invoice-page-dd (pageChanged)="goToRoute($event)" (selectVoucher)="voucherChange($event)"></invoice-page-dd>-->
                     <div id="setting-tab" class="w-100">
-                        <hamburger-menu></hamburger-menu>
+                        <hamburger-menu [pageHeading]="(isMobileScreen ? getPageHeading() : '') "></hamburger-menu>
```

Comment:
do the same here as well like commented in contact.component.html

Reply:
Done